Docs about AWS hubs and CI/CD automation (#717)
2i2c-org/infrastructure: docs/howto/operate/new-hub/index.md

8. Log in to the hub and ensure that the hub works as expected from a user's perspective.
9. Send a link to the hub's Community Representative(s) so they can confirm that it works from their perspective as well.

:::{note}
If you need to deploy a new hub in a AWS cluster, there are some minor additional steps described on this [section](new-hub:aws).
:::

## Automated vs. manual deploys

Some of our infrastructure automatically deploys and updates hubs via GitHub Workflows, while others require manual deploys.
This is changing over time as we automate more things, and is dependent on the cloud provider.

> [](operate:ci-cd)

Reviewer: this seems like an extra addition?

You will find more information about our automation in [](operate:ci-cd)

## Deploying hubs manually

Some of our infrastructure still requires manual deploys.

> There are also situations where you may want to deploy infrastructure manually.

Reviewer: Just a note that I think once we get the AWS automation stuff set up, we should **heavily** discourage deploying anything manually. I feel like every manual step is a large increase in the likelihood of confusion, bottlenecks of information, inconsistent state on the hub vs. the codebase, etc.

8. Log in to the hub and ensure that the hub works as expected from a user's perspective.
9. Send a link to the hub's Community Representative(s) so they can confirm that it works from their perspective as well.

:::{note}
There are some minor additional steps if you need to [deploy a new hub in a AWS cluster](new-hub:aws).
:::

## Automated vs. manual deploys

Some of our infrastructure automatically deploys and updates hubs via GitHub Workflows, while others require manual deploys.
This is changing over time as we automate more things, and is dependent on the cloud provider.

More details about our AWS CI/CD automation lives in [](operate:setup-ci-cd-aws).

## Deploying hubs manually

Some of our infrastructure still requires manual deploys.
There are also situations where you may want to deploy infrastructure manually.
Or situation where some specific steps still need some manual intervention.

The following sections cover how to deploy in these situations:

* [General manual deployment process](operate:manual-deploy)

> * Some [AWS specific manual steps](new-hub:aws) at the time to deploy a new hub for first time

Reviewer 2: Maybe?
```suggestion
* Some [AWS specific manual steps](new-hub:aws) needed to deploy a new hub for first time
```

:::{note}

> You should try to avoid a manual deployments.
> They increase the likelihood of confusion, bottlenecks of information, inconsistent
> states and discrepancies between what is already deployed vs. the codebase, among other
> things.

Reviewer 2: ```suggestion
Manual deploys should be avoided when possible.
They increase the likelihood of confusion, bottlenecks of information, inconsistent
states and discrepancies between what is already deployed vs. the codebase, among other
things.
```
